src/MaintenanceHistoryAPI/Migrations/MaintenanceHistoryContextModelSnapshot.cs

```diff
@@ -1,10 +1,10 @@
 ﻿// <auto-generated />
 using System;
-using MaintenanceHistoryAPI.DataAccess;
 using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Infrastructure;
 using Microsoft.EntityFrameworkCore.Metadata;
 using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Pitstop.MaintenanceHistoryAPI.DataAccess;
```

Comment:
Is dit nog wel nodig als deze al in de globalUsings.cs staat?

Reply:
Is auto generated, hoort bij de migratie